diff --git a/framework/CMakeLists.txt b/framework/CMakeLists.txt
index 3b3a506b..cce5d2fc 100644
--- a/framework/CMakeLists.txt
+++ b/framework/CMakeLists.txt
@@ -43,7 +43,7 @@ usMacroCreateBundle(Framework
   LINK_LIBRARIES ${_link_libraries}
   SOURCES $<TARGET_OBJECTS:absl> ${_framework_srcs}
   PUBLIC_HEADERS ${_framework_public_headers}
-  PRIVATE_HEADERS $<TARGET_PROPERTY:absl,INCLUDE_DIRECTORIES> ${_framework_private_headers}
+  PRIVATE_HEADERS ${_framework_private_headers}
   BINARY_RESOURCES manifest.json
 )
 
